h/services/search_index/service.py

```diff
@@ -2,7 +2,7 @@
 
 from h.presenters import AnnotationSearchIndexPresenter
 from h.services.annotation_read import AnnotationReadService
-from h.tasks import indexer
+from h.tasks import indexer as indexer_tasks
```

Comment:
I like having the explicit "tasks" here, it makes it more clear what we are actually doing below.

Reply:
Yeah, much better :+1: